src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #fff;
+}
+
+:root {
+  --color: #00acdc;
+}
+
+.header {
+  min-width: 1024px;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__icon {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
```

Comment:
Height is 100% by default. You can remove redundant styles. The display property seems redundant as well